README.md

```diff
@@ -1,6 +1,19 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
+
+
+# Read and Write Workflow 
+
+The bootstrap uses the following process to handle incoming EDI:
```

Comment:
I think we need to come up with a way to refer to the bootstrap repository. I use _bootstrap workflow_ most of the time and only use _bootstrap repository_ if I’m talking about the GitHub repository. I see you use _the bootstrap_. I started out using just _Bootstrap_, but Mohan prefers not treating it like a product, so not capitalization. I’m not saying we have to do it as I did, just that it would be good to pick one way.